src/query/formats/src/field_decoder/row_based.rs

```diff
@@ -272,10 +272,15 @@ pub trait FieldDecoderRowBased: FieldDecoder {
                 column.commit_row();
             }
             Err(_) => {
-                return Err(ErrorCode::BadBytes(format!(
-                    "Invalid JSON value: {:?}",
-                    String::from_utf8_lossy(&buf)
-                )));
+                if self.common_settings().disable_json_check {
```

Comment:
if this setting is enabled, we should commit without JSON parse to improve the performance.

Reply:
valid JSON should convert to jsonb format.